benchflow-test-manager/application/src/main/java/cloud/benchflow/testmanager/strategy/selection/ExperimentSelectionStrategy.java

```diff
@@ -0,0 +1,13 @@
+package cloud.benchflow.testmanager.strategy.selection;
+
+/**
+ * @author Jesper Findahl (jesper.findahl@usi.ch)
+ *         created on 2017-04-20
+ */
+public interface ExperimentSelectionStrategy {
```

Comment:
I would not put any logic into the DSL library since these are specific to services.